### PR TITLE
[1LP][RFR] 5.10 Fixes for Containers Tests

### PR DESCRIPTION
**Pull request by jawatts**
{{ pytest: cfme/tests/containers/test_properties.py cfme/tests/containers/test_projects_dashboard.py cfme/tests/containers/test_basic_metrics.py cfme/tests/containers/test_overview_data_integrity.py cfme/tests/containers/test_ssl_providers.py  --use-provider ocp-39-hawk }}

### CODE REVIEW

File: cfme/tests/containers/test_smart_management.py [hunk 1]
```diff
@@ -1,114 +0,0 @@
-import re
```
Comment (jawatts) [inline cfme/tests/containers/test_smart_management.py]: This test is the same as https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/containers/test_tags.py

**Comment by jawatts** [comment]
Failure in PRT is due to:
```/usr/lib64/python2.7/urllib2.py:1200: URLError
URLError
<urlopen error [Errno 113] No route to host>```

Not due to changes in this PR

File: widgetastic_manageiq/__init__.py [hunk 2]
```diff
@@ -630,14 +630,22 @@ def read(self):
 
 
 class NestedSummaryTable(SummaryTable):
-    HEADER_IN_ROWS = "./tbody/tr[1]/td"
-    HEADERS = "./tbody/tr[1]/td/strong"
+    HEADER_IN_ROWS = "./tbody/tr[0]/td"
```
Comment (jawatts) [inline widgetastic_manageiq/__init__.py]: I checked our repo, no other tests are using `NestedSummaryTable`

File: cfme/tests/containers/test_basic_metrics.py [hunk 3]
```diff
@@ -85,11 +85,15 @@ def test_validate_metrics_collection_db(provider,
...
-def test_validate_metrics_collection_provider_gui(provider,
+def test_validate_metrics_collection_provider_gui(appliance, provider,
                                                   enable_capacity_and_utilization,
                                                   reduce_metrics_collection_threshold,
                                                   wait_for_metrics_rollup, soft_assert):
 
+    view = navigate_to(provider, "Details")
+    # Wait for the Utilization drop down to become enabled
+    wait_for(lambda: view.toolbar.monitoring.item_enabled('Utilization'),
+             delay=15, timeout=600, fail_func=appliance.server.browser.refresh)
```
Comment (mshriver) [inline cfme/tests/containers/test_basic_metrics.py]: Shorter delay possible? 2s maybe even?

File: cfme/tests/containers/test_overview_data_integrity.py [hunk 4]
```diff
@@ -47,9 +47,15 @@ def get_api_object_counts(appliance):
...
             # TODO image count? list_templates too high, list_image_stream_images too low
-            # TODO container count? list_container returning different count than MIQ reports
             out[ImageRegistry] += len(provider.mgmt.list_image_registry())
-            out[Container] += len(provider.mgmt.list_container())
+            listed_containers = provider.mgmt.list_container()
+            # Get all container pods
+            count_containers = []
+            for containers in listed_containers:
+                for container in containers:
+                    count_containers.append(container.name)
```
Comment (mshriver) [inline cfme/tests/containers/test_overview_data_integrity.py]: We don't actually need the names, we really just want the count of inner items.

I'd suggest using something like:
```python
count_containers = sum(1 for pod in list_containers for c in pod)
```

**Pull request merged**